test/logging/logging.go

```diff
@@ -159,6 +161,10 @@ func InitializeLogger() {
 		logger = zap.New(zapCore)
 		zap.ReplaceGlobals(logger) // Gets used by klog/glog proxy libraries
 
+		// Set klog/glog verbosities (works with and without proxy libraries)
+		klogLevel := klog.Level(0)
+		klogLevel.Set(fmt.Sprint(Verbosity))
```

Comment:
Verbosity.String()?

Reply:
I accidentally copied an old version, it shouldn't be capitalized.

It's an int and int doesn't have a .String() AFAICT.

Reply:
If it's an int, then just `strconv.Itoa`.